Dockerfile

```diff
@@ -5,11 +5,12 @@ LABEL maintainer="smartlab-dev@mpt.mp.br"
 COPY requirements.txt /app/requirements.txt
 
 RUN apt-get update && \
-    apt-get install -y build-essential uwsgi-plugin-python3 && \
+    apt-get install -y build-essential gcc && \
     pip3 install -r /app/requirements.txt && \
     mkdir -p /var/run/flask && \
     groupadd -r uwsgi && useradd -r -g uwsgi uwsgi && \
     chown -R uwsgi:uwsgi /var/run/flask && \
+    apt-get --purge remove -y gcc && \
```

Comment:
Why are you keeping 'build-essential'?

Reply:
No particular reason. As soon as possible, I'll remove it and test for impacts downstream.